core/src/main/scala/filodb.core/zipkin/Zipkin.scala

```diff
@@ -126,15 +128,19 @@ class ZipkinReporter extends SpanReporter {
     val zipkinHost = Kamon.config().getString(HostConfigKey)
     val zipkinPort = Kamon.config().getInt(PortConfigKey)
 
+    var url = s"$zipkinEndpoint/api/v2/spans"
     if (zipkinEndpoint == null || zipkinEndpoint.trim.isEmpty) {
-      AsyncReporter.create(
-        OkHttpSender.create(s"http://$zipkinHost:$zipkinPort/api/v2/spans")
-      )
-    } else {
-      AsyncReporter.create(
-        OkHttpSender.create(s"$zipkinEndpoint/api/v2/spans")
-      )
+      url = s"http://$zipkinHost:$zipkinPort/api/v2/spans"
```

Comment:
Rather than use a var:

    val url = if (zipkinEndpoint == null || zipkinEndpoint.trim.isEmpty) s"http://$zipkinHost:$zipkinPort/api/v2/spans"
                   else s"$zipkinEndpoint/api/v2/spans"

Reply:
Ok. This is taken care of.